_posts/2021-01-08-2020-review.md

```diff
@@ -0,0 +1,264 @@
+
+A series of virtual events around the project started with a roadmap and
+overview meeting and included a number real world use case examples at scale:
+
+* [State of Trino]({% post_url 2020-05-15-state-of-presto %})
+* [Trino at Pinterest]({% post_url 2020-07-22-presto-summit-pinterest %})
+* [Trino Migration at ARM Treasure Data]({% post_url 2020-07-06-presto-summit-arm-td %})
+* [Trino at Zuora]({% post_url 2020-06-16-presto-summit-zuora %})
+
+Another series of training classes with the project founders was hugely
+successful. It includes very valuable content for any Trino user, from beginners
+to experts, that you should not miss:
+
+* [Advanced SQL in Trino with David]({% post_url 2020-07-15-training-advanced-sql %})
+* [Understanding and Tuning Trino Query Processing with Martin]({% post_url 2020-07-30-training-query-tuning %})
+* [Securing Trino with Dain]({% post_url 2020-08-13-training-security %})
+* [Configuring and Tuning Trino with Dain]({% post_url 2020-08-27-training-performance %})
+
+## Features and code
+
+While everything mentioned is already exciting, the true work is visible in the
+new features and improvements in Trino. It is a long list, but read on. You
+won’t want to miss anything.
+
+### Improvements to ANSI SQL support
+
+A core feature of Trino is the ability to use the same standard SQL for any
+connected data source. These improvements empower all users.
+
+* Variable-precision temporal types, with precision down to picoseconds
+  (10<sup>−12</sup>s). This a very important feature for any time critical
+  systems such as financial transactions processing
+* Correct, and now SQL specification compliant timestamp semantics, making
+  migration of SQL statements from other compliant systems such as many RDBMSs
+  easier
+* Implicit coercions for `INSERT` clause.
+* Support for `RANGE` and `GROUPS`-based window frames
+* More support for various shapes of correlated subqueries
+* Support for `INTERSECT ALL` and `EXCEPT ALL`
+* Experimental support for [recursive queries](/docs/current/sql/select.html?highlight=recursive#with-recursive-clause)
+* Enforcement of `NOT NULL` constraints when inserting data
+* Quantified comparisons (e.g., > ALL (...)) in aggregations
+
+### Other query improvements
+
+A number of other features were added to make querying your data sources with
+Trino even more powerful:
+
+* [T-digest data type](/docs/current/language/types.html#t-digest) and functions
+  for approximate quantile computations
+* Support for setting and reading column comments
+* Numerous new functions including `concat_ws()`, `regexp_count()`,
+  `regexp_position()`, `contains_sequence()`, `murmur3()`,
+  `from_unixtime_nanos()`, `from_iso8601_timestamp_nanos()`,
+  `human_readable_seconds()`, `bitwise` operations, `luhn_check()`,
+  `approx_most_frequent()`, `translate()`, `starts_with()`
+
+## Performance
+
+Trino is already [ludicrously fast](/index.html). But then again, even faster is
```

Comment:
Maybe we could make these the links?
https://trino.io/blog/2020/06/14/dynamic-partition-pruning.html

I would say we should add https://trino.io/blog/2020/08/14/dereference-pushdown.html